pisa-proxy/parser/mysql/src/grammar.y

```diff
@@ -3737,6 +3738,7 @@ non_reserved_keyword -> &'input str:
   | 'JSON'                                             { "JSON" }
   | 'JSON_VALUE'                                       { "JSON_VALUE" }
   | 'KEYRING'                                          { "KEYRING" }
+  | 'KEYS'                                             { "KEYS" }
```

Comment:
`KEYS` is not nonreserved keyword.

Reply:
> `KEYS` is not nonreserved keyword.

https://dev.mysql.com/doc/refman/8.0/en/keywords.html

<img width="491" alt="image" src="https://user-images.githubusercontent.com/5917359/176584339-4b22aa8e-f3c3-4bfa-8fdf-c42c2636751e.png">

Reply:
> `KEYS` is not nonreserved keyword.

https://dev.mysql.com/doc/refman/8.0/en/keywords.html

Reply:
`KEYS` is reversed keyword,  it should not appear in `non_reserved_keyword` rule,

Reply:
> `KEYS` is not nonreserved keyword.

fix done.